server/routes/induction/update/personalInterestsUpdateController.ts

```diff
@@ -0,0 +1,26 @@
+import { NextFunction, Request, RequestHandler, Response } from 'express'
+import PersonalInterestsController from '../common/personalInterestsController'
+
+/**
+ * Controller for the Update of the Personal Interests screen of the Induction.
+ */
+export default class PersonalInterestsUpdateController extends PersonalInterestsController {
+  constructor() {
+    super()
+  }
+
+  getBackLinkUrl(req: Request): string {
+    const { prisonNumber } = req.params
+    return `/plan/${prisonNumber}/view/work-and-interests`
+  }
+
+  getBackLinkAriaText(_req: Request): string {
+    return 'Back to <TODO - check what CIAG UI does here>'
+  }
+
+  submitPersonalInterestsForm: RequestHandler = async (
```

Comment:
The submit method signature just needs to exist at the moment.
Once we understand the form data etc we can flesh out this controller method, which will also mean renaming the args from underscore prefixes (the underscore prefix keeps the linter happy re: unused method args)